datasets/russian_super_glue/README.md

```diff
@@ -0,0 +1,370 @@
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://russiansuperglue.com/
+- **Repository:** https://github.com/RussianNLP/RussianSuperGLUE
+- **Paper:** https://russiansuperglue.com/download/main_article
+- **Leaderboard:** https://russiansuperglue.com/leaderboard/2
+- **Point of Contact:** [More Information Needed]
+
+### Dataset Summary
+
+Modern universal language models and transformers such as BERT, ELMo, XLNet, RoBERTa and others need to be properly
+compared and evaluated. In the last year, new models and methods for pretraining and transfer learning have driven 
+striking performance improvements across a range of language understanding tasks.
+
+
+We offer testing methodology based on tasks, typically proposed for “strong AI” — logic, commonsense, reasoning.
+Adhering to the GLUE and SuperGLUE methodology, we present a set of test tasks for general language understanding
+and leaderboard models.
+
+
+For the first time a complete test for Russian language was developed, which is similar to its English analog.
+Many datasets were composed for the first time, and a leaderboard of models for the Russian language with comparable
+results is also presented.
+
+### Supported Tasks and Leaderboards
+
+Supported tasks, barring a few additions, are equivalent to the original SuperGLUE tasks.
+
+|Task Name|Equiv. to|
+|----|---:|
+|Linguistic Diagnostic for Russian|Broadcoverage Diagnostics (AX-b)|
+|Russian Commitment Bank (RCB)|CommitmentBank (CB)|
+|Choice of Plausible Alternatives for Russian language (PARus)|Choice of Plausible Alternatives (COPA)|
+|Russian Multi-Sentence Reading Comprehension (MuSeRC)|Multi-Sentence Reading Comprehension (MultiRC)|
+|Textual Entailment Recognition for Russian (TERRa)|Recognizing Textual Entailment (RTE)|
+|Russian Words in Context (based on RUSSE)|Words in Context (WiC)|
+|The Winograd Schema Challenge (Russian)|The Winograd Schema Challenge (WSC)|
+|Yes/no Question Answering Dataset for the Russian (DaNetQA)|BoolQ|
+|Russian Reading Comprehension with Commonsense Reasoning (RuCoS)|Reading Comprehension with Commonsense Reasoning (ReCoRD)|
+
+### Languages
+
+All tasks are in Russian.
+
+## Dataset Structure
+
+### Data Instances
+
```

Comment:
Could you please add as Data Instances, a dataset example for each of the tasks?

For example, for task LiDiRus, add the example in https://russiansuperglue.com/tasks/task_info/LiDiRus#Example
```python
{
     'sentence1': "Кошка сидела на коврике.",
     'sentence2': "Кошка не сидела на коврике.",
     'label': 'not_entailment',
     'knowledge': '',
     'lexical-semantics': '',
     'logic': 'Negation',
     'predicate-argument-structure': ''
}
```

Reply:
I've added separate examples for train/dev and test, because the differences aren't always obvious. Also, I decided to sacrifice authenticity for sake of readability and wrapped the examples with exceedingly long text fragments with line breaks, although I'm still on the fence about this. On another note, the examples are specifically after the transformations, so the demonstrated format isn't completely identical to what's actually downloaded (as is the case with the original SuperGLUE). This is the least confusing way, in my opinion, since that's the format the end user is (presumably) going to use, after all.

Reply:
Thanks ! Could you also write explicitly at the beginning of the `Data Instance` section that the test sets are missing labels ?